docs/static/jvm.asciidoc

```diff
@@ -0,0 +1,46 @@
+[float]
+[[ls-jvm]]
+=== Java (JVM) version
+
+{ls} requires one of these versions:
+
+* Java 8
+* Java 11
+* Java 14
+
+Use the
+http://www.oracle.com/technetwork/java/javase/downloads/index.html[official
+Oracle distribution] or an open-source distribution, such as
+http://openjdk.java.net/[OpenJDK].
+
+[float]
+[[check-jvm]]
+==== Check your Java version
+Run the following command:
+
+[source,shell]
+java -version
+
+On systems with Java installed, this command produces output similar to the following:
+
+[source,shell]
+-----
+java version "11.0.1" 2018-10-16 LTS
+Java(TM) SE Runtime Environment 18.9 (build 11.0.1+13-LTS)
+Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.1+13-LTS, mixed mode)
+-----
+
+[float]
+[[java-home]]
+==== `JAVA_HOME`
```

Comment:
@robbavey @karenzone What about windows? 

And we should provide an example of how to set it (e.g. export JAVA_HOME=/path/to/your/java